cli/internal/publish/images/images.go

```diff
@@ -92,8 +123,9 @@ func ProcessDocument(ctx context.Context, c *cloudquery_api.ClientWithResponses,
 		}
 		item := item
 		absFile := ims[listKey{name: item.Name, sum: item.Checksum}][0].absFile
+		contentType := ims[listKey{name: item.Name, sum: item.Checksum}][0].contentType
```

Comment:
This should be item.RequiredHeaders and that should be passed down to uploadFile

Reply:
Something's wrong with the code gen as this is what I see for `RequiredHeaders`:
```go
// RequiredHeaders Required HTTP headers to include for the upload
RequiredHeaders map[string]interface{} `json:"required_headers"`
```

Looks like it should be `RequiredHeaders map[string][]string` (the frontend client looks ok)

Reply:
Interesting but setting each key to a `[]string` should still work?

Reply:
I'm verifying now the runtime type is `map[string][]string` so casting to it will work

Reply:
OK https://github.com/cloudquery/cloudquery/pull/19373/commits/eeb17507c5917915b96b051c07c50a022848cad1 fixes this, tested locally with the EntraID plugin which is currently failing to publish because of this issue:

```
Publishing plugin cloudquery/entraid@v1.0.0 to CloudQuery Hub...
Uploading table schemas...
Uploading docs...
Preparing to upload images...
Uploading images...
Uploading binaries CloudQuery Hub...
- Uploading linux_amd64...
- Uploading linux_arm64...
- Uploading windows_amd64...
- Uploading darwin_amd64...
- Uploading darwin_arm64...
Success!
```